app/src/main/java/org/simple/clinic/home/help/HelpScreenEffectHandler.kt

```diff
@@ -13,12 +13,16 @@ class HelpScreenEffectHandler @AssistedInject constructor(
     private val helpRepository: HelpRepository,
     private val helpSync: HelpSync,
     private val schedulersProvider: SchedulersProvider,
-    @Assisted private val uiActions: HelpScreenUiActions
+    @Assisted private val uiActions: HelpScreenUiActions,
+    @Assisted private val viewEffectHandler: HelpScreenViewEffectHandler
```

Comment:
Just an FYI, we don't have to add the view effect handler here as a dependency. We can directly add the consumer.